src/util/instanceBulkActions.tsx

```diff
@@ -75,8 +75,8 @@ export const pluralize = (item: string, count: number): string => {
     return item;
   }
 
-  if (item.includes("identity")) {
-    return item.replace("identity", "identities");
+  if (item.toLowerCase().includes("identity")) {
+    return item.toLowerCase().replace("identity", "identities");
```

Comment:
This should not mess with the case of the input. When asked to pluralize "Identity", it should not return the lowercase plural, or it has potential to mess up the copy. Maybe a better solution is to add another conditional for the upper case option.